elasticsearch/src/main/java/org/hibernate/search/elasticsearch/util/impl/GsonHttpEntity.java

```diff
@@ -262,6 +316,15 @@ private void hintContentLenght(long contentLenght) {
 		//unnecessarily.
 		boolean flowControlPushingBack = false;
 
+		//Accumulate the content length while it's being produced so we can hint
+		//other components.
+		//This is only used for the case in which we can encode the full message
+		//without ever rotating the active buffer (in one pass), so that we
+		//can calculate the accurate size upfront.
+		//Computing this size after multiple buffers is possible but not helpful,
+		//as we'll already be using the chunked encoding mode.
+		long totalWrittenBytes = 0;
```

Comment:
thanks, I'll remove it. It's just a leftover from a previous approach which didn't work out.